doc/project/js-environments.md

```diff
@@ -47,12 +47,22 @@ For more options of the PhantomJS environment, see
 
 ## <a name="node-on-ubuntu"></a> Node.js on Ubuntu
 
-On Ubuntu, the Node.js command from the [nodejs package](http://packages.ubuntu.com/utopic/nodejs) is called `nodejs` instead of `node` (when installed through the package manager). This will make the Node.js environment fail (since it simply calls `node`).
+Easiest way to handle Node.js versions and installations on Ubuntu (and in Linux systems in general) is to use [nvm](https://github.com/creationix/nvm). All instructions are included.
 
-You have two options to solve this:
+In short it boils down to:
+
+    curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.29.0/install.sh | bash
+    . ~/.nvm/nvm.sh
+
+Add these lines to your ~/.bashrc, ~/.profile, or ~/.zshrc file to have it automatically sourced upon login:
+
+    export NVM_DIR="$HOME/.nvm"
+    [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" # This loads nvm
```

Comment:
I would not repeat these instructions here. They will become obsolete without us knowing. Let people click the link to the `nvm` readme.